src/main/java/org/testng/IConfigurationListener2.java

```diff
@@ -2,9 +2,10 @@
 
 public interface IConfigurationListener2 extends IConfigurationListener {
```

Comment:
Please move this interface contents into its parent interface and have this deprecated.